docs-src/docs/reference/setup-cli.md

```diff
@@ -1,278 +1,212 @@
-# Layer0 Setup (l0-setup) command-line interface reference
-
-The **l0-setup** application is designed to be used with one of several commands: [apply](#apply), [backup](#backup), [destroy](#destroy), [endpoint](#endpoint), [plan](#plan), [restore](#restore), [terraform](#terraform), and [vpc](#vpc). These commands are detailed in the sections below.
-
-##General Usage
-<div class="divTable">
-  <div class="divRow">
-    <div class="divCellNoPadding">**l0-setup** [--version] _command_ [_options_] [_parameters_]</div>
-  </div>
-</div>
+# Layer0 Setup Reference
+The Layer0 Setup application (commonly called **l0-setup**), is used to provision, update, and destroy Layer0 instances.
 
 ---
+## General Usage
+You can use the `-h, --help` command to get genrate information about the `l0-setup` tool:
+
+```
+$ l0-setup -h
```

Comment:
[nit] not sure if we need this output in the doc, it's annoying to re-paste if the `-h` output changes, too